soc/arm/nxp_imx/rt/dts_fixup.h

```diff
@@ -25,7 +25,7 @@
 #define DT_IRQ_ETH1_COMMON			DT_NXP_KINETIS_ETHERNET_402D4000_IRQ_0
 #define DT_IRQ_ETH1_IEEE1588_TMR		DT_NXP_KINETIS_PTP_402D4000_PTP_IRQ_0
 
-#define CONFIG_ENTROPY_NAME			DT_NXP_KINETIS_TRNG_400CC000_LABEL
+#define DT_ENTROPY_NAME			DT_NXP_KINETIS_TRNG_400CC000_LABEL
```

Comment:
Why isn't this `DT_LABEL(DT_INST(0, nxp_kinetis_trng))`

Reply:
just historic, leaving what that part of the define alone.